samples/CppArExample/CppArExample.cpp

```diff
@@ -310,6 +317,45 @@ void CppArExample::createTabletopTestScene()
   changeScene();
 }
 
+// If m_screenToLocationMode is true, create and place a 3D sphere in the scene. Otherwise set the
+// initial transformation based on the screen position.
+void CppArExample::onTouched(QTouchEvent& event)
+{
+  Q_CHECK_PTR(m_arcGISArView);
+
+  const QPoint screenPoint = event.touchPoints().first().lastScreenPos().toPoint();
+
+  // If "screenToLocation" mode is enabled.
+  if (m_screenToLocationMode)
+  {
+    // Get the real world location for screen point from AR view.
+    const Point point = m_arcGISArView->screenToLocation(screenPoint);
+    if (point.isEmpty())
+      return;
+
+    // Get or create graphic overlay
+    Q_CHECK_PTR(m_sceneView);
+    GraphicsOverlay* graphicsOverlay = m_sceneView->graphicsOverlays()->first();
+    if (!graphicsOverlay)
+    {
+      graphicsOverlay = new GraphicsOverlay(this);
+      m_sceneView->graphicsOverlays()->append(graphicsOverlay);
+    }
+    Q_CHECK_PTR(graphicsOverlay);
+
+    // Create and place a graphic at the real world location.
+    SimpleMarkerSceneSymbol* sphere = new SimpleMarkerSceneSymbol(
+          SimpleMarkerSceneSymbolStyle::Sphere, QColor("yellow"), 0.25, 0.25, 0.25,
```

Comment:
Prefer QColor enum constructor when possible.

```suggestion
          SimpleMarkerSceneSymbolStyle::Sphere, QColor(Qt::yellow), 0.25, 0.25, 0.25,
```